scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -1144,6 +1148,30 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
             $scope.closePreview();
             macros.setupShortcuts($scope);
             initEmbedFieldsValidation();
+
+            // init redux
+            const initialState = {
+                isLocked: $scope.isLocked(),
+                isLockedByMe: $scope.isLockedByMe(),
+            };
+
+            function editor(state = initialState) {
```

Comment:
What does this do?

Reply:
that's initial editor store, I have to for state namespacing, `isLocked` shouldn't be part of attachments state

Reply:
What do you mean by **initial** editor reducer? Does the reducer change over time?

Reply:
I mean it should get extended as we port more features to react/redux from angular

Reply:
So it doesn't do anything now, right? I would remove it that case, or at least move it to a separate file and document why is it empty.

Reply:
it's used by attachments component, it just doesn't change